app/common/common-routes.js

```diff
@@ -49,6 +49,14 @@ module.exports = ['$stateProvider', function ($stateProvider) {
                 template: ''
             }
         )
+        .state(
+            {
+                name: '404',
+                url: '/404',
+                controller: require('./auth/404.controller.js'),
```

Comment:
Could we just make this a component instead? especially since it has an empty controller.

Reply:
@crcommons will you check into this in this PR or look into it in a separate one? Not sure if I should test now or after this is addressed.